src/auditlog_tests/models.py

```diff
@@ -120,11 +134,49 @@ class DateTimeFieldModel(models.Model):
     history = AuditlogHistoryField()
 
 
+class ChoicesFieldModel(models.Model):
+    """
+    A model with a CharField restricted to a set of choices.
+    This model is used to test the changes_display_dict method.
+    """
+    RED = 'r'
+    YELLOW = 'y'
+    GREEN = 'g'
+
+    STATUS_CHOICES = (
+        (RED, 'Red'),
+        (YELLOW, 'Yellow'),
+        (GREEN, 'Green'),
+    )
+
+    status = models.CharField(max_length=1, choices=STATUS_CHOICES)
+    multiselect = MultiSelectField(max_length=3, choices=STATUS_CHOICES, max_choices=3)
```

Comment:
Right. If this is added to the tests I would also want the built-in PostgreSQL equivalent/replacement of it (https://docs.djangoproject.com/en/1.10/ref/contrib/postgres/fields/#arrayfield) since it is likely that people store lists that way as well.